shared/src/commonMain/resources/MR/base/strings.xml

```diff
@@ -588,4 +588,13 @@
     <!-- Earned badge modal -->
     <string name="earned_badge_modal_title">Wow! You\'ve reached level %d</string>
     <string name="earned_badge_modal_description">You\'ve earned the %s badge by reaching level %d! Amazing job!</string>
+
+    <!--Paywall-->
+    <string name="paywall_mobile_only_title">Solve unlimited problems with Mobile only plan</string>
+    <string name="paywall_mobile_only_option_1">Access to all tracks</string>
+    <string name="paywall_mobile_only_option_2">Unlimited problems per day in the app</string>
+    <string name="paywall_mobile_only_option_3">1 hint per problem</string>
+    <string name="paywall_mobile_only_buy_btn">Subscribe for $12.00/month</string>
```

Comment:
Please see [question](https://vyahhi.myjetbrains.com/youtrack/issue/ALTAPPS-1115/Shared-Android-Implement-a-Paywall-screen-after-sign-in-sign-up#focus=Comments-74-278754.0-0) about formatting